src/main/util.hpp

```diff
@@ -0,0 +1,19 @@
+#include <map>
+#include <string>
+
+namespace util {
+
+enum Severity {
+    NORMAL,
+    DEBUG,
+    ERROR,
+    WARNING,
+    FATAL,
+};
+
+const char *SeverityText[] = {
```

Comment:
This is minor, but because this is a variable could we call it `severity_text`?

Reply:
I would love to use snake case more than anything. `snake_case > *`

Reply:
Its a const, so shouldn't it be `SCREAMING_SNAKE_CASE`?